### PR TITLE
proposal file structure

### PR DESCRIPTION
**Pull request by axlehner**
first shot proposal. material still in `s-ICC`

### CODE REVIEW

**Comment by wwiecek** [comment]
@axlehner I am okay with whatever file structure works for you. But where should I review these materials? Will you push them into this repository next?

File: README.md [hunk 1]
```diff
@@ -6,3 +6,32 @@ Repository featuring DIL Analytics Manual and its associated documents. To acces
...
+│   │   ├── covariates/ 	#
+│   │	│   ├── examples/   	# Example notebooks for ICC estimation from prior projects and datasets
+│   │   ├── ICC/        	#
+│   │	│   ├── examples/   	# Example notebooks for ICC estimation from prior projects and datasets
+│   │   ├── panel_RCT/      #
+│   │	│   ├── examples/   	# Example code that illustrates power calculations 
+│   │   ├── stratification/ #
+│   │	│   ├── examples/   	# Example code stratification / matching
+│   │   ├── multip_testing/ # MHT corrections
+│   │	│   ├── examples/   	# Example code MHT
```
Comment (luizaandrade) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -6,3 +6,32 @@ Repository featuring DIL Analytics Manual and its associated documents. To acces
...
+## Internal vs. External Documents
+
+Everything that is on the gh-pages branch is on the website and thus external. From there we link to internal documents (on the DIL git) if necessary.
+
+# File Structure
+
+Proposed file structure to host all DIL "educational documents" and important example code from past DIL projects.
+
+```
+gh-pages branch       # DIL Analytics Manual
```
Comment (luizaandrade) [inline README.md:18]:
```suggestion
gh-pages branch       # DIL Analytics Manual
├── Principles of Analytics
├── Coding Guides # Detailed articles with general code snippets
│   |── Power calculations/ 	#
│   │   ├── Covariates
│   │   ├── ICC      	
│   │   ├── Panel RCT
│   │   ├── stratification
│   │   ├── Multiple Hypotheses Testing
├── DIL templates
├── External resources

```

File: README.md [hunk 3]
```diff
@@ -6,3 +6,32 @@ Repository featuring DIL Analytics Manual and its associated documents. To acces
...
+main branch           # Root directory for lab guides
+├── power_calc/       	# High-level document summarizing power calculations @ DIL (including step-by-step guidlines for project specific notebooks)
+│   ├── shiny/       	# An interactive app that showcases all essential examples
+│   ├── examples/     	# Example documents with replication that showcase power calculations from past DIL projects
```
Comment (luizaandrade) [inline README.md:33]:
```suggestion
│   ├── examples/     	# Example documents with replication that showcase power calculations from past DIL projects
│   │   ├── power_calc/
│   │   │   ├── covariates.R
│   │   │   ├── icc.R
│   │   │   ├── panel_rct.R
│   │   │   ├── stratification.R
│   │   │   ├── mht.R
```

**Review by luizaandrade — CHANGES_REQUESTED** [review]
Hi @axlehner , I have made some suggestions based on our chat. Happy to iterate over these.

**Review by luizaandrade — APPROVED** [review]

**Pull request merged**